pylint_django/tests/input/func_noerror_foreign_key_in_non_django_class.py

```diff
@@ -0,0 +1,25 @@
+"""
+Checks that Pylint raise error when a 'ForeignKey' appears in a
```

Comment:
Shouldn't the wording be "doesn't raise an error" ?

Reply:
Thanks for the correction. Updated!